resources/views/dashboard/master.blade.php

```diff
@@ -56,6 +38,12 @@
             $('select').material_select();
           });
         </script>
+
+        <script src = "{!! load_asset('/js/jquery.min.js') !!}"></script>
+        <!-- Materialize Compiled and minified JavaScript -->
+        <script src = "{!! load_asset('/js/materialize.min.js') !!}"></script>
+        <script src = "{!! load_asset('/library/sweetalert/sweetalert.min.js') !!}"></script>
+        <script src = "{!! load_asset('/js/scripts.js') !!}"></script>
         <script src = "{!! load_asset('/js/dashboard.js') !!}"></script>
```

Comment:
**Code Quality - Nitpick**

Why do we still have `dashboard.js` referenced on its own?..Shouldn't it be minified into `scripts.js` ?